bindata/network/openshift-sdn/001-crd.yaml

```diff
@@ -117,6 +117,10 @@ spec:
     type: string
     description: The CIDR range of the overlay network assigned to the node for its pods
     JSONPath: .subnet
+  - name: EgressIPs
```

Comment:
In HostSubnet, you should also print the `EgressCIDRs`, before the `EgressIPs`. (https://github.com/openshift/origin/blob/release-3.11/pkg/printers/internalversion/printer.go#L66)

Reply:
PR updated